<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>745</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
/* QWidget ----------------------------------------------------------------

--------------------------------------------------------------------------- */
QWidget {
/*  background-color: #021701;*/
background-color: #000d00;

  border: 0.25px solid #950101;
  padding: 0px;
  color: #F0F0F0;
  selection-background-color: #1464A0;
  selection-color: #F0F0F0;
}

QWidget:disabled {
  background-color: #130101;
  color: #787878;
  selection-background-color: #14506E;
  selection-color: #787878;
}

QWidget::item:selected {
  background-color: #130101;
}

QWidget::item:hover {
  background-color: #130101;
  color: #32414B;
}




/* QTextBrowser -----------------------------------------------------------

--------------------------------------------------------------------------- */
QTextBrowser {
  background-color: #290101;
  border: 1.5px solid #bd0000;
  color: #c9c5c5;
  font: 75 16pt &quot;Arial&quot;;
  border-radius: 4px;
}

QTextBrowser:disabled {
  background-color: #290101;
  border: 1.5px solid #bd0000;
  color: #c9c5c5;
  font: 75 16pt &quot;Arial&quot;;
  border-radius: 4px;
}

QTextBrowser:hover, QTextBrowser:!hover, QTextBrowser:selected, QTextBrowser:pressed {
  border: 1.5px solid #bd0000;
}

/* QTextLabel ----------------------------------------------------------

--------------------------------------------------------------------------- */

QTextLabel {
  background-color: #290101;
  border: 1.5px solid #bd0000;
  color: #c9c5c5;
  font: 75 16pt &quot;Arial&quot;;
  border-radius: 4px;
}

QTextLabel:disabled {
  background-color: #290101;
  border: 1.5px solid #bd0000;
  color: #c9c5c5;
  font: 75 16pt &quot;Arial&quot;;
  border-radius: 4px;
}

QTextLabel:hover, QTextBrowser:!hover, QTextBrowser:selected, QTextBrowser:pressed {
  border: 1.5px solid #bd0000;
}




/* QPushButton ----------------------------------------------------------

--------------------------------------------------------------------------- */

QPushButton {
    color: #c9c5c5;
	font: 75 16pt &quot;Arial&quot;;

    border: 2px solid #007a0c;
    border-radius: 20px;
    border-style: outset;
    background: qradialgradient(
        cx: 0.2, cy: -0.3, fx: 0.2, fy: -0.2,
        radius: 1.35, stop: 0 #199612, stop: 1 #000000
        );

	
    padding: 5px;
    }

QPushButton:hover {
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #000000, stop: 1 #199612
        );
    }

QPushButton:pressed {
    border-style: inset;
    background: qradialgradient(
        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
        radius: 1.35, stop: 0 #000000, stop: 1 #007a0c
        );
    }



</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>260</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> color: #c9c5c5;
	
font: 75 italic 14pt &quot;Comic Sans MS&quot;;

    border: 0.25px solid #810303;
    border-radius: 10px;
    border-style: outset;
    background: qradialgradient(
        cx: 0.2, cy: -0.3, fx: 0.2, fy: -0.2,
        radius: 1.35, stop: 0 #199612, stop: 1 #000000
        );

	
    padding: 2px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>365</x>
      <y>260</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> color: #c9c5c5;
	
font: 75 italic 14pt &quot;Comic Sans MS&quot;;

    border: 0.25px solid #810303;
    border-radius: 10px;
    border-style: outset;
    background: qradialgradient(
        cx: 0.2, cy: -0.3, fx: 0.2, fy: -0.2,
        radius: 1.35, stop: 0 #199612, stop: 1 #000000
        );

	
    padding: 2px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>260</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> color: #c9c5c5;
	
font: 75 italic 14pt &quot;Comic Sans MS&quot;;

    border: 0.25px solid #810303;
    border-radius: 10px;
    border-style: outset;
    background: qradialgradient(
        cx: 0.2, cy: -0.3, fx: 0.2, fy: -0.2,
        radius: 1.35, stop: 0 #199612, stop: 1 #000000
        );

	
    padding: 2px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>365</x>
      <y>180</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> color: #c9c5c5;
	
font: 75 italic 14pt &quot;Comic Sans MS&quot;;

    border: 0.25px solid #810303;
    border-radius: 10px;
    border-style: outset;
    background: qradialgradient(
        cx: 0.2, cy: -0.3, fx: 0.2, fy: -0.2,
        radius: 1.35, stop: 0 #199612, stop: 1 #000000
        );

	
    padding: 2px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>180</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> color: #c9c5c5;
	
font: 75 italic 14pt &quot;Comic Sans MS&quot;;

    border: 0.25px solid #810303;
    border-radius: 10px;
    border-style: outset;
    background: qradialgradient(
        cx: 0.2, cy: -0.3, fx: 0.2, fy: -0.2,
        radius: 1.35, stop: 0 #199612, stop: 1 #000000
        );

	
    padding: 2px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>180</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> color: #c9c5c5;
	
font: 75 italic 14pt &quot;Comic Sans MS&quot;;

    border: 0.25px solid #810303;
    border-radius: 10px;
    border-style: outset;
    background: qradialgradient(
        cx: 0.2, cy: -0.3, fx: 0.2, fy: -0.2,
        radius: 1.35, stop: 0 #199612, stop: 1 #000000
        );

	
    padding: 2px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>251</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgba(255, 255, 255,2);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../GUI materials/image.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> color: #c9c5c5;
	
font: 75 italic 14pt &quot;Comic Sans MS&quot;;

    border: 0.25px solid #810303;
    border-radius: 10px;
    border-style: outset;
/*background-color:#042402 ;*/
    

	
    padding: 2px;


/*background: qradialgradient(
        cx: 0.2, cy: -0.3, fx: 0.2, fy: -0.2,
        radius: 1.35, stop: 0 #199612, stop: 1 #000000
        );</string>
    </property>
    <property name="text">
     <string>Pitch:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>295</x>
      <y>180</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> color: #c9c5c5;
	
font: 75 italic 14pt &quot;Comic Sans MS&quot;;

    border: 0.25px solid #810303;
    border-radius: 10px;
    border-style: outset;
/*background-color:#042402 ;*/
    

	
    padding: 2px;


/*background: qradialgradient(
        cx: 0.2, cy: -0.3, fx: 0.2, fy: -0.2,
        radius: 1.35, stop: 0 #199612, stop: 1 #000000
        );</string>
    </property>
    <property name="text">
     <string>Roll:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>180</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> color: #c9c5c5;
	
font: 75 italic 14pt &quot;Comic Sans MS&quot;;

    border: 0.25px solid #810303;
    border-radius: 10px;
    border-style: outset;
/*background-color:#042402 ;*/
    

	
    padding: 2px;


/*background: qradialgradient(
        cx: 0.2, cy: -0.3, fx: 0.2, fy: -0.2,
        radius: 1.35, stop: 0 #199612, stop: 1 #000000
        );</string>
    </property>
    <property name="text">
     <string>Rotate:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>285</x>
      <y>260</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> color: #c9c5c5;
	
font: 75 italic 14pt &quot;Comic Sans MS&quot;;

    border: 0.25px solid #810303;
    border-radius: 10px;
    border-style: outset;
/*background-color:#042402 ;*/
    

	
    padding: 2px;


/*background: qradialgradient(
        cx: 0.2, cy: -0.3, fx: 0.2, fy: -0.2,
        radius: 1.35, stop: 0 #199612, stop: 1 #000000
        );</string>
    </property>
    <property name="text">
     <string>Hight:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> color: #c9c5c5;
	
font: 75 italic 14pt &quot;Comic Sans MS&quot;;

    border: 0.25px solid #810303;
    border-radius: 10px;
    border-style: outset;
/*background-color:#042402 ;*/
    

	
    padding: 2px;


/*background: qradialgradient(
        cx: 0.2, cy: -0.3, fx: 0.2, fy: -0.2,
        radius: 1.35, stop: 0 #199612, stop: 1 #000000
        );</string>
    </property>
    <property name="text">
     <string>Temp:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>260</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> color: #c9c5c5;
	
font: 75 italic 14pt &quot;Comic Sans MS&quot;;

    border: 0.25px solid #810303;
    border-radius: 10px;
    border-style: outset;
/*background-color:#042402 ;*/
    

	
    padding: 2px;


/*background: qradialgradient(
        cx: 0.2, cy: -0.3, fx: 0.2, fy: -0.2,
        radius: 1.35, stop: 0 #199612, stop: 1 #000000
        );</string>
    </property>
    <property name="text">
     <string>Speed:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> color: #c9c5c5;
	
font: 75 italic 14pt &quot;Comic Sans MS&quot;;

    border: 0.25px solid #810303;
    border-radius: 10px;
    border-style: outset;
    background: qradialgradient(
        cx: 0.2, cy: -0.3, fx: 0.2, fy: -0.2,
        radius: 1.35, stop: 0 #199612, stop: 1 #000000
        );

	
    padding: 2px;</string>
    </property>
    <property name="text">
     <string>Disconnected</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> color: #c9c5c5;
	
font: 75 italic 14pt &quot;Comic Sans MS&quot;;

    border: 0.25px solid #810303;
    border-radius: 10px;
    border-style: outset;
/*background-color:#042402 ;*/
    

	
    padding: 2px;


/*background: qradialgradient(
        cx: 0.2, cy: -0.3, fx: 0.2, fy: -0.2,
        radius: 1.35, stop: 0 #199612, stop: 1 #000000
        );</string>
    </property>
    <property name="text">
     <string>Current status:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> color: #c9c5c5;
	
font: 75 italic 14pt &quot;Comic Sans MS&quot;;

    border: 0.25px solid #810303;
    border-radius: 10px;
    border-style: outset;
	background-color:rgba(2,23,1,5) ;
    

	
    padding: 2px;


/*background: qradialgradient(
        cx: 0.2, cy: -0.3, fx: 0.2, fy: -0.2,
        radius: 1.35, stop: 0 #199612, stop: 1 #000000
        );</string>
    </property>
    <property name="text">
     <string>Enter IP:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>100</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> color: #c9c5c5;
	
font: 75 italic 14pt &quot;Comic Sans MS&quot;;

    border: 0.25px solid #810303;
    border-radius: 10px;
    border-style: outset;
	background-color:rgba(2,23,1,5) ;
    

	
    padding: 2px;


/*background: qradialgradient(
        cx: 0.2, cy: -0.3, fx: 0.2, fy: -0.2,
        radius: 1.35, stop: 0 #199612, stop: 1 #000000
        );</string>
    </property>
    <property name="text">
     <string>Enter PORT:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>90</y>
      <width>91</width>
      <height>46</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>100</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> color: #c9c5c5;
	
font: 75 italic 14pt &quot;Comic Sans MS&quot;;

    border: 0.25px solid #810303;
    border-radius: 10px;
    border-style: outset;
    background: qradialgradient(
        cx: 0.2, cy: -0.3, fx: 0.2, fy: -0.2,
        radius: 1.35, stop: 0 #199612, stop: 1 #000000
        );

	
    padding: 2px;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>100</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> color: #c9c5c5;
	
font: 75 italic 14pt &quot;Comic Sans MS&quot;;

    border: 0.25px solid #810303;
    border-radius: 10px;
    border-style: outset;
    background: qradialgradient(
        cx: 0.2, cy: -0.3, fx: 0.2, fy: -0.2,
        radius: 1.35, stop: 0 #199612, stop: 1 #000000
        );

	
    padding: 2px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> color: #c9c5c5;
	
font: 75 italic 14pt &quot;Comic Sans MS&quot;;

    border: 0.25px solid #810303;
    border-radius: 10px;
    border-style: outset;
	background-color:rgba(2,23,1,5) ;
    

	
    padding: 2px;


/*background: qradialgradient(
        cx: 0.2, cy: -0.3, fx: 0.2, fy: -0.2,
        radius: 1.35, stop: 0 #199612, stop: 1 #000000
        );</string>
    </property>
    <property name="text">
     <string>Current motion:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>340</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> color: #c9c5c5;
	
font: 75 italic 14pt &quot;Comic Sans MS&quot;;

    border: 0.25px solid #810303;
    border-radius: 10px;
    border-style: outset;
    background: qradialgradient(
        cx: 0.2, cy: -0.3, fx: 0.2, fy: -0.2,
        radius: 1.35, stop: 0 #199612, stop: 1 #000000
        );

	
    padding: 2px;</string>
    </property>
    <property name="text">
     <string>Static</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>330</y>
      <width>121</width>
      <height>46</height>
     </rect>
    </property>
    <property name="text">
     <string>Disconnect</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>745</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
